docs/content/https/acme.md

```diff
@@ -606,7 +606,7 @@ docker run -v "/my/host/acme:/etc/traefik/acme" traefik
 
 _Optional, Default=2160_
 
-The `certificatesDuration` option defines the certificates' duration in hours.
+The `certificatesDuration` option defines how long the certificates issued by your certificate authority will last.
```

Comment:
Maybe I would remove this sentence, and use the explanation already existing below.

Something like:

`certificatesDuration` is used to calculate two durations:
- `Renew Period`: the period before the end of the certificate duration, during which the certificate should be renewed.
- `Renew Interval`: the interval between renew attempts.

| Certificate Duration | Renew Period      | Renew Interval          |
|----------------------|-------------------|-------------------------|
| >= 1 year            | 4 months          | 1 week                  |
| >= 90 days           | 30 days           | 1 day                   |
| >= 7 days            | 1 day             | 1 hour                  |
| >= 24 hours          | 6 hours           | 10 min                  |
| < 24 hours           | 20 min            | 1 min                   |

It defaults to `2160` (90 days) to follow Let's Encrypt certificates' duration.

...Examples

Reply:
Agree, i think your explanation gave the exact role for `certificatesDuration`. Should I change it? Btw Thanks!

Reply:
Sure